fed/_private/constants.py

```diff
@@ -25,7 +25,7 @@
 
 KEY_OF_TLS_CONFIG = "TLS_CONFIG"
 
-KEY_OF_CROSS_SILO_MESSAGE_CONFIG = "CROSS_SILO_MESSAGE_CONFIG"
+KEY_OF_CROSS_SILO_COMM_CONFIG_DICT = "CROSS_SILO_COMM_CONFIG_DICT"
```

Comment:
As discussed in https://github.com/ray-project/rayfed/pull/140#discussion_r1263257404, "COMM" may be misunderstood to "common"

Reply:
But `Cross-Silo Message` is not very right. Cross-Silo Communication is very common in privacy preserving computation, but `Message` is strange. And other users ask me why not `cross_silo_comm`.

Reply:
A short name could be not explicit since so many objects share same short name. Users can read the docstring to get known the explicit meaning. And we do not have a param like `CrossSiloCustomConfig`.